src/renderer/pages/Staking/Operations/components/Confirmation/Confirmation.tsx

```diff
@@ -23,7 +23,7 @@ import { useMultisigTx } from '@renderer/entities/multisig';
 import { RewardsDestination, WalletType } from '@renderer/shared/core';
 import type { Account, Asset, Explorer } from '@renderer/shared/core';
 import { AssetFiatBalance } from '@renderer/entities/price/ui/AssetFiatBalance';
-import { SelectedValidatorsModal } from '../Modals/SelectedValidatorsModal/SelectedValidatorsModal';
+import { SelectedValidatorsModal } from '../../../../../entities/staking/ui/SelectedValidatorsModal/SelectedValidatorsModal';
```

Comment:
Add component to `entitites/staking/ui/index.ts` and `entitites/staking/index.ts` and import it from there